src/de/deserializer.rs

```diff
@@ -52,6 +52,8 @@ pub enum Error {
     InsufficientData,
 }
 
+impl serde::ser::StdError for Error {}
```

Comment:
Doesn't this need to be gated with a `std` feature?

Reply:
No.
https://github.com/serde-rs/serde/blob/a6a8a334f7b6a3a031f03a74f3c6ca7dac1240d3/serde/src/std_error.rs#L37-L42